homeassistant/components/rfxtrx/__init__.py

```diff
@@ -409,6 +409,25 @@ def assumed_state(self):
         """Return true if unable to access real state of entity."""
         return True
 
+    @property
+    def unique_id(self):
+        """Return unique identifier of remote device."""
+        return f"{slugify(self._event.device.type_string.lower())}_{slugify(self._event.device.id_string.lower())}"
+
+    @property
+    def device_info(self):
```

Comment:
There's no config entry for this integration. Then `device_info` doesn't work.

Reply:
Working on that. But sure I'll drop it from this pull.